src/main/java/com/gmail/val59000mc/scoreboard/ScoreboardManager.java

```diff
@@ -16,6 +16,7 @@
 import com.gmail.val59000mc.threads.UpdateScoreboardThread;
 import com.gmail.val59000mc.utils.TimeUtils;
 import com.gmail.val59000mc.utils.VersionUtils;
+import me.clip.placeholderapi.PlaceholderAPI;
```

Comment:
Having this import will stop the plugin from loading on servers that don't have placeholder API installed.